README.md

```diff
@@ -128,6 +128,14 @@ Plans:
   standard100   Standard Tier, 100 DTUs
 ```
 
+## Provision a service
+
+```console
+$ svcat provision quickstart-wordpress-mysql-instance \
```

Comment:
@arschles Can I get your opinion on verbs?

* `svcat provision` vs. `svcat create instance`
* `svcat deprovision` vs. `svcat delete instance`
* `svcat bind` vs. `svcat create binding`
* `svcat unbind` vs. `svcat delete binding`

I am not suggesting that we do this, but want to make sure we know why we didn't go down a more "kubectl-esque" path. Is it better to be as close as possible to kubectl with the verbs, or be more domain specific?

Specifically `deprovision` made me think about this, because `delete` makes it clear that its a destructive operation.

_NOTE: sync is a great example of a purely domain specific (non-CRUD) verb. I'm just wondering about these other ones_

Reply:
I'd go with the domain specific ones